src/config/config.rs

```diff
@@ -41,25 +42,45 @@ pub fn config_validator<'a>() -> V::Structure<'a> {
         command_validator()))
 }
 
-fn find_config_path(work_dir: &PathBuf) -> Option<(PathBuf, PathBuf)> {
+fn find_config_path(work_dir: &PathBuf, show_warnings: bool)
+    -> Option<(PathBuf, PathBuf)>
+{
+    fn maybe_print_warning(dir: &Path) {
+        if dir.join("vagga.yml").exists() {
+            writeln!(&mut stderr(), "There is vagga.yml file in the {:?}, \
+                    possibly it is a typo. \
+                    Correct configuration file name is vagga.yaml",
+                dir).ok();
+        }
+    }
+    
     let mut dir = work_dir.clone();
     loop {
         let fname = dir.join(".vagga/vagga.yaml");
         if fname.exists() {
```

Comment:
> we can add a warning if there is vagga.yml but not vagga.yaml

I think this is reasonable to print warning if only incorrect file name is present.